test/runtests.jl

```diff
@@ -131,5 +131,5 @@ using Base.Test, Nulls
     @test Nulls.T(Any) == Any
     io = IOBuffer()
     show(io, Union{Float64, Null})
-    @test String(take!(io)) == "?Float64"
+    @test String(take!(io)) == "Union{Nulls.Null, Float64}"
```

Comment:
Don't worry about that; just make the test say `@test String(take!(io)) in ("Union{Nulls.Null, Float64}", "Union{Float64, Nulls.Null}")`. There's a ton of logic that goes into the Union type sorting that's in 0.7 only.